neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/impl/StringLoggerToJBossLoggingAdaptor.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.ogm.datastore.neo4j.impl;
+
+import org.neo4j.helpers.collection.Visitor;
+import org.neo4j.kernel.impl.util.StringLogger;
+import org.neo4j.kernel.logging.LogMarker;
+
+import org.hibernate.ogm.datastore.neo4j.logging.impl.Log;
+import org.hibernate.ogm.datastore.neo4j.logging.impl.LoggerFactory;
+
+/**
+ * Implements Neo4J's StringLogger and write the content to JBoss Logging.
+ * Regular messages are sent as debug, user messages as info, exception messages as error
+ * and long messages as separated calls to logMessage.
+ * This follows Neo4J's {@code LogbackService} implementation.
+ *
+ * @author Emmanuel Bernard &lt;emmanuel@hibernate.org&gt;
+ */
+public class StringLoggerToJBossLoggingAdaptor extends StringLogger {
```

Comment:
If you want to go down that path, also `isDebugEnabled()` should be overridden (delegating to `log`) so one can actually make the messages visible.